en/companion_computer/README.md

```diff
@@ -36,9 +36,8 @@ PX4 can be used with computers that can be configured to communicate via MAVLink
 
 A small subset of possible alternatives are listed below:
 
-- Raspberry Pi
-- Odroid
-- Tegra K1
+- [ModalAI VOXL 2](https://docs.modalai.com/voxl2-external-flight-controller/)
+- [NXP NavQPlus](https://nxp.gitbook.io/navqplus)
```

Comment:
1. Why delete RaPi? It's probably the most broadly supported companion out there - and actually has docs https://docs.px4.io/main/en/companion_computer/pixhawk_rpi.html

2. We have less information about Odroid, but it is referenced here https://docs.px4.io/main/en/ros/offboard_control.html#on-board-processor along with some other options like NuC. Can you pull out the options you want from that doc, put them here, and link from that doc to here?

3. I don't know anything about TegraK1 - you're just deleting it because it seems defunct?

4. Very happy to add VOXL2 - there is evidence that this is a supported use case - https://docs.modalai.com/voxl2-external-flight-controller/#voxl-2-as-companion-computer-px4-sw-setup

5. The link for NXP NavQPlus looks interesting, but pretty unhelpful for working with PX4. We can add it, but having some actual instructions for use with PX4 would be nice.

Reply:
I agree. No reason to remove RPi.